heft-plugins/heft-jest-plugin/src/JestPlugin.ts

```diff
@@ -488,6 +505,11 @@ export default class JestPlugin implements IHeftTaskPlugin<IJestPluginOptions> {
     await Promise.resolve();
 
     if (pendingTestRuns.size > 0) {
+      // set env variable if not already set
+      if (!process.env.NODE_ENV) {
+        process.env.NODE_ENV = 'test';
```

Comment:
This should do a `this._nodeEnvSet = true`?

Reply:
you are right, also if statement should be different. I've fixed it